builtin/k8s/platform.go

```diff
@@ -120,6 +120,18 @@ func (p *Platform) Deploy(
 	step.Update("Kubernetes client connected to %s with namespace %s", config.Host, ns)
 	step.Done()
 
+	step = sg.Add("Validating cluster...")
+
+	nodes, err := clientSet.CoreV1().Nodes().List(ctx, metav1.ListOptions{})
+	if err != nil {
+		step.Update("Unable to read node list, assuming cluster healthy")
+	} else {
+		if nodes.Size() == 0 {
```

Comment:
I thought about it over the evening and I'm unsure if this is right as well. Seems like a cluster could be configured to spawn nodes only when needed, and thusly would fail this check.

Reply:
Yeah its probably safe.